### ISSUE TITLE
I can't train successfully by using command in rexgen_direct/core_wln_global/notes.txt

### ISSUE THREAD

**Issue by sstary** [connorcoley/rexgen_direct#16]
I just changed bucket_size to [10,15,20,25,30,35,40,45,50,55] cause hardware limitation and there is no other difference.
But when I use "python nntrain_direct.py --train ../data/train.txt.proc --hidden 300 --depth 3 --save_dir model-300-3-direct",
I get this:
Model size: 757K
Acc@10: 0.0020, Acc@20: 0.0030, Param Norm: 55.96, Grad Norm: 32601341764211.89
Acc@10: 0.0020, Acc@20: 0.0030, Param Norm: 74.00, Grad Norm: 1186378715784512768.00
Acc@10: 0.0030, Acc@20: 0.0030, Param Norm: 83.09, Grad Norm: inf
Acc@10: 0.0010, Acc@20: 0.0040, Param Norm: 83.23, Grad Norm: inf
……

Grad Norm get "inf" in the third Iteration, I want to know if this is right? And if not, how can i fix this?

**Comment by sstary** [comment]
I made a mistake here:
loss = tf.nn.sigmoid_cross_entropy_with_logits(labels=tf.to_float(flat_label), logits=flat_score)
Now my model is ok.
